requirements/common_constraints.txt

```diff
@@ -30,3 +31,10 @@ elasticsearch<7.14.0
 # So we need to pin it globally, for now.
 # Ticket for unpinning: https://github.com/openedx/edx-lint/issues/407
 importlib-metadata<7
+
+# Cause: https://github.com/openedx/event-tracking/pull/290
+# event-tracking 2.4.1 upgrades to pymongo 4.4.0 which is not supported on edx-platform.
+# We will pin event-tracking to do not break existing installations
+# This can be unpinned once https://github.com/openedx/edx-platform/issues/34586
+# has been resolved and edx-platform is running with pymongo>=4.4.0
+event-tracking<2.4.1
```

Comment:
[clarification] It looks like the linked [issue](https://github.com/openedx/edx-platform/issues/34586) and [corresponding PR](https://github.com/openedx/edx-platform/pull/34675), merged last week. Is this constraint still needed?

Reply:
This was automatically added by make upgrade, so as long as it is upstream it will continue to be copied into this file even if I removed it now.